docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -0,0 +1,124 @@
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*, and follow the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA] guide.
+
+[discrete]
+[[cspm-use-instance-role]]
+==== Option 2 - Use default instance role
+From https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2]:
+
+When you create an IAM role using the IAM console, the console creates an instance profile automatically and gives it the same name as the role to which it corresponds.
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
+
+[discrete]
+[[cspm-use-keys-directly]]
+==== Option 3 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details, refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
+
+[discrete]
+[[cspm-use-temp-credentials]]
+==== Option 4 - Use temporary security credentials
+You can configure temporary security credentials in AWS to last for a specified duration. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+Because temporary security credentials are short term, once they expire, you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
+
+NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details, refer to AWS' https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Temporary Security Credentials] documentation.
+
+You can use the AWS CLI to generate temporary credentials. For example, you could use the following command if you have MFA enabled:
+
+[source,console]
+----------------------------------
+`sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --duration-seconds 129600 --token-code 123456`
```

Comment:
I think the backticks are redundant here, because the `source` block is already rendering the line as code